src/main/java/org/terracotta/build/plugins/AngelaPlugin.java

```diff
@@ -49,17 +51,20 @@ public AngelaPlugin(JvmPluginServices jvmPluginServices) {
   @Override
   public void apply(Project project) {
     project.getPlugins().apply(JvmEcosystemPlugin.class);
+    project.getPlugins().apply(CustomHostPlugin.class);
```

Comment:
Angela shouldn't be applying the custom host plugin... it should react to it (if it's enabled).